prototypeCode/roverCode/prototype/wirelessCommunication.ino

```diff
@@ -7,7 +8,11 @@
 // GLOBAL VARIABLES //
 EthernetUDP Udp;
 byte MAC_ADDRESS[] = {0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED};
-IPAddress IP(192, 168, 1, 4);
+int UPD_PORT = 8888;
+IPAddress IP(192, 168, 1, 51);
+char packetBuffer[UDP_TX_PACKET_MAX_SIZE];
+int x = 0;
+int y = 0;
 
 void initializeWirelessCommunication()
 {
```

Comment:
You need to put some commands in this function. Additionally you can't have all of your subsequent function declarations inside of this function. C++ doesn't allow declaring a function inside of another function.